ember.md

```diff
@@ -115,11 +117,69 @@ set(someObject, 'isUpdated', true);
 get(someObject, 'isUpdated'); //true
 ```
 
+
+<a name="data"></a>
+## Data
+
+<a name="data--data-binding"></a>
+### 3.1 Data Binding
+
+Any new code written by ASH should only contain one-way data-binding. Third-party addons using two-way data-binding is ok, but be cautious and conscious of the side effects it can have.
+
+> Why? Two-way data-binding can have unexpected side effects; data flowing one way keeps things more predictable. For example: if you pass the same data to 2 components and they both are allowed to change it. DDAU allows the parent to arbitrate those changes and resolve conflicts. Otherwise component A can make changes you are not expecting in component B. This is not just a matter of 2 compoents, you can make a change to a child that propagates through the whole app.
+
+```hbs
+{{!--Bad--}}
+{{!--Parent-component.hbs--}}
```

Comment:
lowercase p

Reply:
@ASH-Anthony Fixed the casing is this good to go now?